README.md

```diff
@@ -42,7 +42,8 @@ pools:
     pm: static
     pm.max_children = 20
 php_ini_values:
-  apc.enabled: 1
+  apc.enabled:
+    - 1
```

Comment:
because depending on what var you are setting multiple values are allowed and are merged by php into one var